tests/template_test.py

```diff
@@ -10,12 +10,12 @@ def bake_in_temp_dir(cookies, *args, **kwargs):
     finally:
         rmtree(str(result.project_path))
 
-def test_bake_project_default_adapter_name(cookies):
+def test_bake_project_adapter_name(cookies):
     """bake and test against default values of cookiecutter.json, adapter_name, and top level files created"""
-    with bake_in_temp_dir(cookies) as result:
+    with bake_in_temp_dir(cookies,extra_context={ "adapter_name": "MyAdapter" }) as result:
         assert result.exit_code == 0
         assert result.exception is None
-        assert result.context["adapter_name"] == "My Adapter"
+        assert result.context["adapter_name"] == "MyAdapter"
         assert result.context["adapter_name"] != "123444"
```

Comment:
Nit: we probably don't need the line 19 assert that the adapter_name isn't 123444 since we assert what it is set to right about it